cypress/integration/inventory/moving-items.spec.js

```diff
@@ -0,0 +1,23 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import testTypes from '../../support/dictionary/testTypes';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+
+describe('inventory: moving items', () => {
+  beforeEach('navigates to Inventory', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C15185 verifies moving multiple items between holding', { tags: [testTypes.smoke] }, () => {
```

Comment:
do we have a chance to prepare special test data to this test?

Reply:
As I can see, we need to add new holding. It seems difficult and test duration will very increase.